aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -10,7 +10,7 @@
     /* global Cl, describe, it, expect */
 
     // ########################################################################
-    describe('cl.events.js', function () {
+    describe('cl.events.js:', function () {
```

Comment:
We need a colon to make the sentence more universal - for example: "cl.events.js: Cl.events._handler returns undefined if direction is "previous" and month is 1"